Makefile

```diff
@@ -20,6 +20,7 @@ android: $(BUILDDIR)/android/tun2socks.aar
 $(BUILDDIR)/android/tun2socks.aar: $(GOMOBILE)
 	mkdir -p "$(BUILDDIR)/android"
 	$(ANDROID_BUILD_CMD) -o "$@" $(IMPORT_PATH)/$(ROOT_PKG)/outline/tun2socks $(IMPORT_PATH)/$(ROOT_PKG)/outline/shadowsocks
+	unzip "$@" 'jni/*' -d $(BUILDDIR)/android
```

Comment:
Why are you uncompressing the aar file?

Reply:
Maybe not in this PR, but I think we should use the aar directly in the future. [Intra already referenced the aar directly](https://github.com/Jigsaw-Code/Intra/blob/master/Android/app/build.gradle#L136-L139), but I don't know whether Cordova allows you to do that.

Reply:
@daniellacosse Can you check whether we need to unzip again?

Based on the new fixed build.gradle, it seems we refer to directly to the .aar file, not the extracted files.